File: java/program/Mamalia.java
public class Mamalia extends MakhlukHidup {
    private int jumlahKaki;
    private int beratBadan;
    private String warnaRambut;

    public Mamalia(String nama, String namaLatin, int usiaMaksimal,
                   int jumlahKaki, int beratBadan, String warnaRambut) {
        super(nama, namaLatin, usiaMaksimal);
        this.jumlahKaki = jumlahKaki;
        this.beratBadan = beratBadan;
        this.warnaRambut = warnaRambut;
    }

    public int getJumlahKaki() { return jumlahKaki; }
    public int getBeratBadan() { return beratBadan; }
    public String getWarnaRambut() { return warnaRambut; }

    public void setJumlahKaki(int jumlahKaki) { this.jumlahKaki = jumlahKaki; }
    public void setBeratBadan(int beratBadan) { this.beratBadan = beratBadan; }
    public void setWarnaRambut(String warnaRambut) { this.warnaRambut = warnaRambut; }

    @Override
    public void displayInfo() {
        System.out.println("--- DATA MAMALIA ---");
        super.displayInfo();
        System.out.println("Jumlah Kaki      : " + jumlahKaki);
        System.out.println("Berat Badan      : " + beratBadan + " kg");
        System.out.println("Warna Rambut     : " + warnaRambut);
    }
}

File: python/program/Mamalia.py
# File: Mamalia.py

# Import kelas induknya
from MakhlukHidup import MakhlukHidup

class Mamalia(MakhlukHidup):
    # Constructor
    def __init__(self, nama="", nama_latin="", usia_maksimal=0, jumlah_kaki=0, berat_badan=0, warna_rambut=""):
        # Panggil constructor dari kelas induk
        super().__init__(nama, nama_latin, usia_maksimal)
        self.__jumlah_kaki = jumlah_kaki
        self.__berat_badan = berat_badan
        self.__warna_rambut = warna_rambut

    # Setter dan Getter khusus Mamalia
    def set_jumlah_kaki(self, jumlah):
        self.__jumlah_kaki = jumlah
        
    def get_jumlah_kaki(self):
        return self.__jumlah_kaki

    def set_berat_badan(self, berat):
        self.__berat_badan = berat

    def get_berat_badan(self):
        return self.__berat_badan

    def set_warna_rambut(self, warna):
        self.__warna_rambut = warna

    def get_warna_rambut(self):
        return self.__warna_rambut

    # Override method display_info dari induk
    def display_info(self):
        print("--- DATA MAMALIA ---")
        super().display_info()  # Panggil display_info dari induk
        print(f"Jumlah Kaki      : {self.get_jumlah_kaki()}")
        print(f"Berat Badan      : {self.get_berat_badan()} kg")
        print(f"Warna Rambut     : {self.get_warna_rambut()}")